eth/protocols/snap/sync.go

```diff
@@ -2771,7 +2771,7 @@ func (s *Syncer) onHealState(paths [][]byte, value []byte) error {
 	if len(paths) == 1 {
 		var account types.StateAccount
 		if err := rlp.DecodeBytes(value, &account); err != nil {
-			return nil
+			return nil // TODO: Shouldn't this return the error? / @holiman
```

Comment:
cc @karalabe ?

Reply:
I think returning an error will drop the remote peer

Reply:
```suggestion
			return nil // Returning the error here would drop the remote peer
```